src/main.rs

```diff
@@ -66,26 +67,40 @@ fn run() -> Result<()> {
     let config: Config = toml::from_str(&toml_doc)
         .wrap_err("Unable to read toml document or find the rpy.tool configuration in it")?;
     let py_config = config.tool.rpy;
-    let python = project_root.join(Path::new(&py_config.interpreter));
-    let src_root = project_root.join(Path::new(
-        &py_config.source_root.unwrap_or_else(|| ".".to_string()),
-    ));
+    let raw_interpreter = env::var("RPY_INTERPRETER").unwrap_or(py_config.interpreter);
+
+    let interpreter = if raw_interpreter.contains('/') {
```

Comment:
superminornit I think https://doc.rust-lang.org/stable/std/path/constant.MAIN_SEPARATOR.html or the equivale str thing could be used here.

Reply:
I originally went looking for that constant and then deliberately changed my mind.
We don't currently support windows, but if/when we do you almost certainly want to either support both forward and backslash or _only_ forward slash since the file we're reading from wouldn't vary along with the OS.